agent/lib/kontena/agent.rb

```diff
@@ -82,6 +82,10 @@ def supervise_launchers
     end
 
     def supervise_network_adapter
+      @supervisor.pool(
```

Comment:
I "think" the doc says that you should not create pool inside a worker that is part of a pool?

![confusing](https://media4.giphy.com/media/3oz8xZvvOZRmKay4xy/200.gif)